src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/UploadEnvelopeDocumentsTaskTest.java

```diff
@@ -30,6 +32,7 @@
     private static final String CONTAINER_1 = "container-1";
     private static final String CONTAINER_2 = "container-2";
     private static final String ZIP_FILE_NAME = "zip-file-name";
+    private static final long MINIMUM_ENVELOPE_AGE = 20;
```

Comment:
seconds ? may be `MINIMUM_ENVELOPE_AGE_IN_SECS` ?

Reply:
in minutes ;)